<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>591</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Данные</string>
    </property>
    <widget class="QPushButton" name="requestPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>571</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить запрос</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>571</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Панель управления</string>
    </property>
    <widget class="QPushButton" name="removePushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="statusDatabaseConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
